modules/oe_webtools_media/tests/src/FunctionalJavascript/MediaSourceWebtoolsTest.php

```diff
@@ -40,10 +40,12 @@ class MediaSourceWebtoolsTest extends MediaSourceTestBase {
    *   The widget human name.
    * @param string $service
    *   The webtools service associated with the widget.
+   * @param string $svg_filename
+   *   The filename of svg.
```

Comment:
We need to rename this. Can we use something generic without the file extension?

Reply:
thumbnail_filename or icon_filename, something like that.